src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -2,11 +2,27 @@
 
 
 public class Task5 {
+
     public static double calculateA(double x, double y, double z) {
-        return 0;
+
+
+        double a;
+        a = (2 * (Math.sin(x - (Math.PI / 6))) * calculateB(z))  / (0.5 + Math.pow(Math.sin(y), 2));
+        return a;
     }
 
     private static double calculateB(double z) {
-        return 0;
+        double b;
+        b = 1 + Math.pow(z,2) / (3 + Math.pow(z,2) / 5);
```

Comment:
In most cases it's better to extract duplicated calculations into separate variable.
Please extract `Math.pow(z, 2)`